src/main/configService.ts

```diff
@@ -33,17 +56,49 @@ export default class ConfigService extends EventEmitter {
          * Getter and setter config listeners. 
          */
         ipcMain.on('config', (event, args) => {
-            const [fn, key] = args;
-
-            if (fn === 'get') {
-                const value = this.get(key);
-                console.log('[Config Service] Get from config store:', key, value);
-                event.returnValue = value;
-            } else
-            if (fn === 'set') {
-                const value = args[2];
-                this.set(key, value);
-                console.log('[Config Service] Set in config store:', key, value);
+            switch (args[0]) {
+                case 'get': {
+                    const value = this.get(args[1]);
+                    event.returnValue = value;
+                    return;
+                }
+
+                case 'set': {
+                    const [_, key, value] = args;
+                    if (this.configValueChanged(key, value)) {
```

Comment:
Am I dumb or is this condition missing an !?

Reply:
You're right. I don't know how I missed that. :|